apps/prairielearn/src/ee/pages/institutionAdminAdmins/institutionAdminAdmins.ts

```diff
@@ -0,0 +1,89 @@
+import { Router } from 'express';
+import asyncHandler = require('express-async-handler');
+import error = require('@prairielearn/error');
+import { flash } from '@prairielearn/flash';
+import {
+  loadSqlEquiv,
+  queryAsync,
+  queryValidatedRows,
+  runInTransactionAsync,
+} from '@prairielearn/postgres';
+
+import {
+  InstitutionAdminAdmins,
+  InstitutionAdminAdminsRowSchema,
+} from './institutionAdminAdmins.html';
+import { getInstitution } from '../../lib/institution';
+import { parseUidsString, selectUserByUid } from '../../../lib/user';
+
+const router = Router({ mergeParams: true });
+const sql = loadSqlEquiv(__filename);
+
+router.get(
+  '/',
+  asyncHandler(async (req, res) => {
+    // TODO: authenticate the the user can access this institution.
+    // TODO: do the same on other institution admin pages as well.
```

Comment:
This is currently done at a parent router. That should be updated to allow institution admins to access these pages.

Reply:
This is no longer relevant; we now use a `selectAndAuthz...` function instead of middleware.